README.rst

```diff
@@ -6,26 +6,20 @@ Numba
    :target: https://gitter.im/numba/numba?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
    :alt: Gitter
 
```

Comment:
Put the logo here?

Reply:
Going to skip this for now, but we can do this in a future PR.